### DISCUSSION TITLE
problem with contribution

### DISCUSSION THREAD

**Discussion by r0yal3858** [community/community#58181] [New to GitHub]
for my collage project i contributed to a repository of my friend's. due to a mismatch in the email of my local repository and the github profile the contribution is not reflected in the repository but the commits are under my name from the local repository. i have changed the local repository name matching to my github profile username and email too but it didnt get updated

**Comment by the-officialjosh** [comment]
f the contributions from your local repository are not reflected in your friend's repository due to a mismatch in email, changing the local repository name and matching it with your GitHub profile username and email might not be sufficient to update the contributions retroactively.

GitHub uses the email associated with the commits to attribute contributions to users. To resolve this issue, you should ensure that the email used in your local repository's commits matches the email associated with your GitHub account at the time the commits were made.

If you've already made the necessary changes to your local repository's email configuration, you can try pushing the commits again to see if they get properly attributed to your GitHub account. However, please note that this will only work for future contributions and won't retroactively update previous commits.

If the issue persists, you might consider reaching out to GitHub support for further assistance in resolving the mismatched contribution attribution problem.
